### PR TITLE
added weka easyconfig

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b92976c08b107fd05e7c for a full test report.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.6, AMD Opteron(tm) Processor 6380, Python 2.6.6
See https://gist.github.com/f2ba27430b10d9457204 for a full test report.

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'WEKA'
+version = '3.6.12'
+versionsuffix = '-Java-1.7.0_80'
```
Comment (boegel) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb]: @pescobar: please avoid copy-pasting the Java version

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://prdownloads.sourceforge.net/weka/']
+sources = ['%s-%s.zip' % (name.lower(), version.replace('.','-'))]
+
+dependencies = [('Java', '1.7.0_80')]
+
+sanity_check_paths = {
+    'files': ["weka.jar"],
+    'dirs': [""]
```
Comment (boegel) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb]: replace with `[]`

File: easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+description = """ Weka is a collection of machine learning algorithms for data mining tasks. 
+ The algorithms can either be applied directly to a dataset or called from your own Java code. 
+ Weka contains tools for data pre-processing, classification, regression, clustering, 
+ association rules, and visualization. It is also well-suited for developing new machine 
+ learning schemes."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://prdownloads.sourceforge.net/weka/']
+sources = ['%s-%s.zip' % (name.lower(), version.replace('.','-'))]
```
Comment (boegel) [inline easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_80.eb]: space after `,`

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6c5ffbd2d49966c8ccbf for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/be467e0106debf283e73 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/978ed783bdb8f12e6a5a for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2ed1c81f8141d0001a7e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**